test/fork/LockupDynamic.t.sol

```diff
@@ -412,5 +416,44 @@ abstract contract LockupDynamic_Fork_Test is Fork_Test {
         YieldMode actualYieldMode = lockupDynamic.getRebasingAssetConfiguration(rebasingAsset);
         YieldMode expectedYieldMode = YieldMode.CLAIMABLE;
         assertEq(uint8(actualYieldMode), uint8(expectedYieldMode));
+
+        /*//////////////////////////////////////////////////////////////////////////
+                                CLAIM REBASING ASSET YIELD
+        //////////////////////////////////////////////////////////////////////////*/
+
+        // Only run the claim tests if the rebasing asset is USDB. That's because WETH distributes yield very
+        // differently from USDB.
+        bool isAssetUSDB = keccak256(abi.encodePacked(rebasingAsset.symbol())) == keccak256(abi.encodePacked("USDB"));
+        if (isAssetUSDB) {
+            // Mint some tokens to the lockupDynamic.
+            changePrank({ msgSender: rebasingAsset.bridge() });
+            rebasingAsset.mint(address(lockupDynamic), 1000e18);
+
+            // Add yield to the rebasing asset.
+            address l2Reporter =
+                address(uint160(rebasingAsset.REPORTER()) + uint160(0x1111000000000000000000000000000000001111));
```

Comment:
Because:
- It will be different on mainnet
- Its an EOA address which may be changed by them

So to avoid changing it, I am querying it from the contract address itself.

Reply:
ok, fair